### DISCUSSION TITLE
Advantages of using racecar over ruby-kafka?

### DISCUSSION THREAD

**Discussion by sandipsubedi** [zendesk/racecar#239] [Q&A] [answered]
I am trying to add a kafka gem on my rails app and I saw `racecar` and `ruby-kafka`. After reading this article: https://medium.com/zendesk-engineering/introducing-racecar-fdb5178bcafb, it seems like `Racecar` is just simpler than `ruby-kafka`.

After reading this article: https://laptrinhx.com/racecar-v2-a-new-foundation-3312594259/, it seems like Racecar is a better option.
<img width="1532" alt="Screen Shot 2021-03-22 at 9 38 12 PM" src="https://user-images.githubusercontent.com/11358435/112084512-f04da200-8b56-11eb-9aa0-d936ca097029.png">

 Is there any other advantage of using racecar over `ruby-kafka`? Any thoughts/comments?

**Comment by dasch** [comment, marked as answer]
If in doubt, use Racecar. It's more opinionated, and sets very sane defaults. It also takes care of a bunch of things you'd otherwise have to do yourself, such as reading config, emitting stats, etc.
